python/ray/experimental/dag/class_node.py

```diff
@@ -47,13 +72,14 @@ def __init__(self, actor: ClassNode, method_name: str):
         self._options = {}
 
     def _bind(self, *args, **kwargs):
+        kwargs_to_resolve = {
+            "method_name": self._method_name,
+            "parent_class_node": self._actor,
+            "prev_class_method_call": self._actor._last_call,
```

Comment:
what's this used for?

Reply:
it's to build lineage of class method calls in ray dag, so a ClassNode as parent can spawn multiple ClassMethodNodes in a chain